idd/Energy+.idd.in

```diff
@@ -19197,6 +19200,408 @@ RoomAirSettings:UnderFloorAirDistributionExterior,
+      \maximum 1.0
+  A3, \field RoomAir:Node:AirflowNetwork:AdjacentSurfaceList Name
+      \type object-list
+      \object-list RoomAirNodeSurfaceLists
+  A4, \field RoomAir:Node:AirflowNetwork:InternalGains Name
+      \type object-list
+      \object-list RoomAirNodeGains
+  A5; \field RoomAir:Node:AirflowNetwork:HVACEquipment Name
+      \type object-list
+      \object-list RoomAirNodeHVACEquipment     
+      
+ RoomAir:Node:AirflowNetwork:AdjacentSurfaceList, 
+       \extensible:1
+       \min-fields 2
+  A1,  \field Name
+       \type Alpha
+       \reference RoomAirNodeSurfaceLists       
+  A2 , \field Surface 1 Name
+       \begin-extensible
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A3 , \field Surface 2 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A4 , \field Surface 3 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A5 , \field Surface 4 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A6 , \field Surface 5 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A7 , \field Surface 6 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A8, \field Surface 7 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A9, \field Surface 8 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A10, \field Surface 9 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A11, \field Surface 10 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A12, \field Surface 11 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A13, \field Surface 12 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A14, \field Surface 13 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A15, \field Surface 14 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A16, \field Surface 15 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A17, \field Surface 16 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A18, \field Surface 17 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A19, \field Surface 18 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A20, \field Surface 19 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A21, \field Surface 20 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+  A22; \field Surface 21 Name
+       \type object-list
+       \object-list AllHeatTranSurfNames
+
+RoomAir:Node:AirflowNetwork:InternalGains,
+      \extensible:4
+      \min-fields 5
+      \memo define the internal gains that are associated with one particular RoomAir:Node
+  A1, \field Name
+      \type Alpha
+      \reference RoomAirNodeGains
+  A2, \field Internal Gain Object 1 Type
```

Comment:
@lgu1234 I know it is late in the game to say this, but this object will be difficult to maintain and difficult to use. When a new internal gain object is added, the developer of that object likely won't know that it needs to be added here - both in code and IDD.  @Myoldmopar Perhaps we need to start a list of dependencies in the developer guide?